src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -8,6 +8,8 @@ import { BidFormFragmentContainer as BidForm } from "Apps/Auction/Components/Bid
 import { LotInfoFragmentContainer as LotInfo } from "Apps/Auction/Components/LotInfo"
 import { AppContainer } from "Apps/Components/AppContainer"
 import { trackPageViewWrapper } from "Apps/Order/Utils/trackPageViewWrapper"
+import { track } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
```

Comment:
import { AnalyticsSchema } from 'Artsy/Analytics'

Reply:
@damassi Briefly, can you elaborate on the "why?" here? Generally attempting to avoid global imports? Taking a look at src/Artsy/Analytics/index.ts, pretty sure they are structurally equivalent. Looks like this is some relevant context: https://github.com/artsy/reaction/pull/2405